### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by AlmazYarullin**
Check this PR: https://github.com/Pumpka-s/task-tracker/pull/6

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -24,7 +25,7 @@ def update
     if @comment.update(comment_params)
       redirect_to @comment.task, notice: 'Comment was successfully updated.'
     else
-      redirect_to @comment.task, alert: 'Comment was not updated'
+      redirect_to @comment.task, alert: @comment.errors.first.full_message
```
Comment (DmitryBarskov) [inline app/controllers/comments_controller.rb]: You should just do `render :edit`

File: app/views/comments/_form.html.slim [hunk 2]
```diff
@@ -1,11 +1,8 @@
-= simple_form_for comment, url: task_comment_path(task_id: comment.task.id)  do |form|
-  / = render '/common/errors', model: @task
+= simple_form_for [task, comment]  do |form|
+  = render '/common/errors', model: comment
```
Comment (DmitryBarskov) [inline app/views/comments/_form.html.slim]: If you use simple_form the errors are shown without this line

**Review by DmitryBarskov — CHANGES_REQUESTED** [review]

File: app/controllers/comments_controller.rb (hunk 1 above)
Comment (AlmazYarullin) [reply app/controllers/comments_controller.rb]: Fixed

File: app/views/comments/_form.html.slim (hunk 2 above)
Comment (AlmazYarullin) [reply app/views/comments/_form.html.slim]: Fixed

**Review by DmitryBarskov — APPROVED** [review]

**Pull request closed without merging**